Makefile

```diff
@@ -170,6 +170,7 @@ build-docs:
 	cd docs && \
 	while read p; do \
 		(git checkout $${p} && npm install && VUEPRESS_BASE="/$${p}/" npm run build) ; \
+		git stash ; \
```

Comment:
Do we call `git stash pop` sometime later? if not, isn't `git checkout .` better?

Reply:
What does the `.` do? I haven't used this feature.

Reply:
It undoes unstaged local modifications

Reply:
thank you!!